bin/garden-init

```diff
@@ -146,9 +146,19 @@ if ! "$debootstrap" "${debootstrapArgs[@]}"; then
 fi
 echo "$epoch" > "$targetDir/garden-epoch"
 
+if [ -v PKG_DIR ]; then
```

Comment:
`[ -v` is a bash-ism. Please use at least `[[ -v` to make it explicit or stick to `[ "$var" ]`.

Reply:
`[ "$var" ]` won't work because we have `set -u`
and the `-v` syntax seems clearer / more legible than some `[ -n "${var+x}" ]` parameter expansion magic
but yeah, using `[[ ... ]]` is probably a good idea :+1: